persistence-modules/spring-boot-persistence-mongodb/src/test/java/com/baeldung/projection/MongoTemplateProjectionUnitTest.java

```diff
@@ -0,0 +1,224 @@
+package com.baeldung.projection;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.data.mongodb.core.query.Query;
+
+import com.baeldung.projection.model.InStock;
+import com.baeldung.projection.model.Inventory;
+import com.baeldung.projection.model.Size;
+
+@SpringBootTest
+public class MongoTemplateProjectionUnitTest extends AbstractTestProjection {
+
+    @BeforeEach
+    void setup() throws Exception {
+
```

Comment:
As noted in another class, avoid blank lines after an opening brace. All the methods in this class begin with a blank line.